specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/agreement.json

```diff
@@ -67,215 +64,274 @@
           }
         ],
         "responses": {
           "200": {
-            "description": "OK. The request has succeeded.",
+            "description": "A billing account.",
             "schema": {
               "$ref": "#/definitions/Agreement"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "Default response",
             "schema": {
-              "$ref": "./types.json#/definitions/ArmError"
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
             }
           }
         },
-        "tags": [
-          "Agreements"
-        ],
         "x-ms-examples": {
-          "agreementsGet": {
-            "$ref": "./examples/agreementsGet.json"
+          "AgreementByName": {
+            "$ref": "./examples/agreementByName.json"
           }
         }
       }
     }
   },
   "definitions": {
     "Agreement": {
+      "description": "An agreement.",
+      "type": "object",
       "allOf": [
         {
           "$ref": "./types.json#/definitions/ArmResource"
         }
       ],
-      "description": "An agreement.",
       "properties": {
         "properties": {
-          "$ref": "#/definitions/AgreementProperties",
-          "description": "The properties of an agreement.",
+          "description": "The properties of a(n) Agreement",
+          "allOf": [
+            {
+              "$ref": "#/definitions/AgreementProperties"
+            }
+          ],
           "x-ms-client-flatten": true
         }
       },
-      "type": "object"
+      "x-ms-azure-resource": true
     },
     "AgreementListResult": {
-      "description": "Result of listing agreements.",
+      "description": "A container for a list of resources",
+      "type": "object",
       "properties": {
         "nextLink": {
           "description": "The link (url) to the next page of results.",
-          "readOnly": true,
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "value": {
-          "description": "The list of agreements.",
+          "description": "The list of resources.",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/Agreement"
           },
           "readOnly": true,
-          "type": "array"
+          "x-ms-identifiers": [
+            "id"
+          ]
         }
       }
     },
     "AgreementProperties": {
-      "description": "The properties of an agreement.",
+      "description": "An agreement.",
+      "type": "object",
       "properties": {
         "acceptanceMode": {
           "description": "The mode of acceptance for an agreement.",
           "enum": [
+            "Unknown",
             "ClickToAccept",
             "ESignEmbedded",
-            "ESignOffline"
+            "ESignOffline",
+            "Implicit",
+            "Offline",
+            "PhysicalSign"
```

Comment:
What's the difference between Offline and PhysicalSign?

Reply:
I don't know. @ramkd12 @amberbhargava  to comment?

Reply:
These are the descriptions of acceptance mode

![image](https://github.com/Azure/azure-rest-api-specs/assets/29668917/62f4c149-893b-4cb1-a8dc-72d39a23e569)